packages/yii-dev-panel-sdk/src/Helper/collectorsTotal.ts

```diff
@@ -4,41 +4,41 @@ import {CollectorsMap} from '@yiisoft/yii-dev-panel-sdk/Helper/collectors';
 export const getCollectedCountByCollector = (collector: CollectorsMap, data: DebugEntry): number | undefined => {
     switch (collector) {
         case CollectorsMap.AssetCollector:
-            return Number(data.asset?.bundles?.total);
+            return Number(data.summary[CollectorsMap.AssetCollector]?.asset?.bundles?.total);
```

Comment:
Also types should be changed https://github.com/yiisoft/yii-dev-panel/blob/fe57f80991bec40e85fef82843c64d8f23a67d25/packages/yii-dev-panel-sdk/src/API/Debug/Debug.ts#L10